<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SigninDialog</class>
 <widget class="QDialog" name="SigninDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>561</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>555</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255)</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>78</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#3e94c9;&quot;&gt;--------------------------------------------------------------------------------------&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/main_title.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>531</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'华文隶书'; font-size:24pt; font-weight:600; color:#2684cb;&quot;&gt;北京市计量检测科学研究院&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>451</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>华文隶书</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;华文隶书&quot;;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#2684cb;&quot;&gt;用户注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>47</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000</string>
       </property>
       <property name="text">
        <string>姓名:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="real_name_lineEdit">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#000000</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>姓名</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>47</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000</string>
       </property>
       <property name="text">
        <string>密码:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#000000</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>用户密码</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>47</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000</string>
       </property>
       <property name="text">
        <string>重复密码:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="check_password_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>347</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>347</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000
</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::PasswordEchoOnEdit</enum>
       </property>
       <property name="placeholderText">
        <string>确认用户密码</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>47</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string>用户名:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_lineEdit">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>用户名</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>47</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000</string>
       </property>
       <property name="text">
        <string>电话：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="telephone_lineEdit">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#000000</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>电话</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>47</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000</string>
       </property>
       <property name="text">
        <string>部门：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="department_lineEdit">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>19</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#000000</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>部门</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="btn_confirm">
       <property name="styleSheet">
        <string notr="true">background-color: #B4CDCD;</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_cancel">
       <property name="styleSheet">
        <string notr="true">background-color: #B4CDCD;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
